rs/sns/governance/canister/governance.did

```diff
@@ -295,6 +295,7 @@ type Governance = record {
 };
 
 type Timers = record {
+  requires_periodic_tasks : opt bool;
```

Comment:
This seems like a smell - like we are exposing API details unnecessarily due to underlying data structures.  The API ought to be separated from the internals.  

I guess this is fine since we can remove an opt bool later, once we separate API types out in the SNS crate, but we should make a note somewhere if possible so that we actually do it when it becomes possible.

Reply:
Okay, will add a comment in a follow up PR.

Reply:
Will be addressed in https://github.com/dfinity/ic/pull/2264